tools/cloud-init-hotplugd

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
```

Comment:
Let's add a comment or breadcrumb here about how this daemon plays a role in proxying udev events to cloud-init devel hotplug command calls. So others can understand some context of how this daemon is being used if this is the first file they find on a system.

Reply:
Done